delfin/api/v1/storages.py

```diff
@@ -103,6 +102,22 @@ def create(self, req, body):
             msg = _('Failed to sync resources for storage: %(storage)s. '
                     'Error: %(err)s') % {'storage': storage['id'], 'err': e}
             LOG.error(msg)
+
+        try:
+            # Trigger Performance monitoring
+            capabilities = self.driver_api.get_capabilities(
+                context=ctxt, storage_id=storage['id'])
+            # validate capabilities
+            validation.validate_capabilities(capabilities)
+            # trigger performance monitoring
+            _create_performance_monitoring_task(ctxt, storage['id'],
+                                                capabilities)
+        except Exception as e:
+            # Unexpected error occurred, while performance monitoring.
+            msg = _('Failed to trigger performance monitoring for storage: '
```

Comment:
how about changing message like  'Failed to create  performance monitoring task for this storage '

Reply:
Updated as per suggestion